recipes/cytoolz/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.7.5" %}
+
+package:
+  name: cytoolz
+  version: {{ version }}
+
+source:
+  fn: cytoolz-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cytoolz/cytoolz-{{ version }}.tar.gz
+  md5: 15fbc1c56fca80cbca0e89f37c93d432
+
+build:
+  number: 0
+  script: python setup.py install --record=record.txt
```

Comment:
Yeah, unfortunately `setuptools` is being pulled in by default when `python` is installed. So, even if you only use `setuptools` when it is found, it will be found. We plan on changing this behavior in the future. For now, unless something never touches `setuptools`, just assume it does and try.